test/mocks/index.ts

```diff
@@ -0,0 +1,218 @@
+import nock from "nock";
+import { expect } from "chai";
+import { numberPurchasableByAccount } from "../../src/actions/sale/numberPurchasableByAccount";
+import { Mintlist, SaleContractConfig } from "../../src/types";
+import { Sale } from "../../src/contracts/types";
+import { SalePhase } from "../../src/types";
+import "mocha";
+import {
+  BigNumber,
+  CallOverrides,
+  ContractTransaction,
+  ethers,
+  Signer,
+} from "ethers";
+import { getMintlist } from "../../src/actions/sale/getMintlist";
+import {
+  getSaleData,
+  getSaleStatus,
+  purchaseDomains,
+} from "../../src/actions/sale";
+import { Block, BlockTag, Provider } from "@ethersproject/providers";
+import { SalePhases } from "../../src/constants";
+
+export const feb23TsSeconds = 1677173138;
+export const feb24TsSeconds = 1677264314;
+
+export type Mocks = {
```

Comment:
Should all these types be individual types instead? And why prefix each with "Mocks"? If we have a definition for what a `SaleConfig` or similar looks like, then we can use that in both the real code and in testing